Public/Src/Tools/Execution.Analyzer/Args.cs

```diff
@@ -594,6 +600,12 @@ private static void WriteHelp()
 
             writer.WriteLine("");
             WriteCopyFilesAnalyzerHelp(writer);
+
+            writer.WriteLine("");
+            WriteXLGToDBHelp(writer);
+
+            writer.WriteLine("");
+            WriteDominoInvocationHelp(writer);

```

Comment:
I'd leave these out of the help text for now since we don't really intend on them being discoverable

Reply:
Ill leave it in there commented out with a todo in front of it. 

---
In reply to: [306093506](https://github.com/microsoft/BuildXL/pull/636#discussion_r306093506) [](ancestors = 306093506)